sonata-project/block-bundle/3.9/config/packages/sonata_block.yaml

```diff
@@ -0,0 +1,5 @@
+framework:
+    templating:
+        engines: ['twig']
+sonata_block:
+    default_contexts: [cms]
```

Comment:
`cms` context is never used anywhere, and it was renamed to `sonata_page_bundle` for SonataPageBundle (see https://stackoverflow.com/questions/22798670/what-is-context-in-sonata-block-bundle). So, i think `cms` should be removed from `default_contexts` here.